app/api/migrations/migrations/57-add_system_key_translations/system_keys.csv

```diff
@@ -0,0 +1,15 @@
+key, optionalValue
+"Services"
+"Loading"
+"OCR PDF"
+"In OCR queue"
+"Unsupported OCR language"
+"OCR error"
+"Could not be processed"
+"OCR"
+"OCR completed"
+"Last updated"
+"This will process the document to recognize it's text. The original file will be added as a supporting file."
+"The OCR engine couldn't read the document due to a malformed format. Last updated"
+"This will enable the Optical Character Recognition (OCR) functionality for PDF documents. This service will recognize text inside images, such as scanned documents and photos, and convert it into machine-readable text data."
+"When activated, this will enable administrator and editor users to send PDF documents to the OCR service."
```

Comment:
@Zasa-san I think these last lines are good candidates to use a shorter key and the long text as value.

Reply:
Done!